Maca aporte5

Macarena12345 opened the pull request:
<!-- Saludos, si usted esta leyendo esto, significa que quiere colaborar con el proyecto aportando quizás mejoras o información que encontró mientras utilizo el mismo, es por eso que queremos agradecerle que se tome su tiempo por hacer lo antes mencionado. -->

<!-- 
	Utilice la siguiente forma de títulos-
•	Si es una mejora en la base de datos [DB] titulo
•	Si es una mejora de estilos [CSS] titulo
•	Si es una mejora en el código [HTML] titulo
Si utiliza varias mejoras, no especifique nada, solo el titulo.

-->
CSS y HTML
<!-- Necesitamos el que titulo del tema sea lo mas representativo posible. -->

##### ¿QUÉ CAMBIO PROPONE?:
<!--
	Agregue un pequeño texto que explique cual es el bug que desea reportar o cual es la situación de mejora que le gustaría que apliquemos. 
-->
Deseo agregar un pie de pagina que contenga informacion sobre quien creo el sitio, donde se creo, y las redes sociales para que el usuario pueda ingresar a las mismas. Para esto modifique el css y el html 



##### NOTAS EXTRA:
<!--
Cualquier información que pueda ayudar a los desarrolladores a identificar y solucionar el problema debe ser puesta aquí.
-->

##### BRANCH(ES):
<!-- 
Especifique la(s) rama(s) afectada(s) por este problema: master, 0.x, 1.x, u otra rama. 
Reemplace el valor de master por la rama correspondiente.
-->

master



##### OTRAS PERSONALIZACIONES:
<!-- 
En caso de que se produzca alguna mejora en el código, avísennos, caso contrario, no es necesario que usted escriba nada aquí. (solo para programadores)
-->




<!-- ------------------------- THE END ------------------------------
 Gracias nuevamente por su colaboración, atenderemos su tema lo antes posible. 
-->

Macarena12345 committed:
Insertando el pie de pagina

Inserte el pie de pagina que contiene los nombres de los creadores y
donde fue creada, tambien tiene las redes sociales. Pie de pagina insertado
en galeria.html y en somos_nosotros.html

Macarena12345 committed:
Agregando estilo al pie de pagina

Agregue estilo al pie de pagina para hacerlo visualmente mas lindo.
Los archivos modificados son galeria.css y style.css

RainTrap reviewed (changes requested):
Agregar una linea en blanco al final de los archivo style.css y galeria.css. En el archivo galeria.html agregar la dirección de correo al href de la linea 200.

Macarena12345 committed:
Modificando el pie

Agregue el mail en el pie de pagina a las paginas galeria.html y contacto.html

Macarena12345 committed:
Modificando el estilo

Agregue una linea al final del codigo para evitar inconvenientes en
el futuro. Los archivos modificados son galeria.css style.css

RainTrap reviewed (approved):
Gracias por realizar los cambios solicitados, el código esta bien.
